apps/backoffice-v2/src/domains/business-reports/fetchers.ts

```diff
@@ -1,28 +1,50 @@
-import { z } from 'zod';
 import { apiClient } from '@/common/api-client/api-client';
 import { Method } from '@/common/enums';
 import { handleZodError } from '@/common/utils/handle-zod-error/handle-zod-error';
+import { z } from 'zod';
 
 export const BusinessReportSchema = z
   .object({
+    riskScore: z.number(),
     report: z.object({
       reportFileId: z.string(),
+      reportId: z.string(),
     }),
+    createdAt: z.string(),
   })
   .optional();
 
-export const fetchBusinessReports = async ({
+export type TBusinessReport = z.infer<typeof BusinessReportSchema>;
+
+export const fetchLatestBusinessReport = async ({
   businessId,
   reportType,
 }: {
   businessId: string;
-  reportType: 'MERCHANT_REPORT_T1' & (string & {});
+  reportType: 'MERCHANT_REPORT_T1' | ('ONGOING_MERCHANT_REPORT_T1' & (string & {}));
```

Comment:
The parenthesis are wrong.
```suggestion
  reportType: ('MERCHANT_REPORT_T1' | 'ONGOING_MERCHANT_REPORT_T1') & (string & {});
```